StatusMonitor/FlightAssistEvent.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace EddiEvents
+{
+    class FlightAssistEvent : Event
+    {
+        public const string NAME = "Flight assist";
+        public const string DESCRIPTION = "Triggered when flight assist is toggled";
+        public const string SAMPLE = null;
+        public static Dictionary<string, string> VARIABLES = new Dictionary<string, string>();
+
+        static FlightAssistEvent()
+        {
+            VARIABLES.Add("status", "Whether flight assist is on");
```

Comment:
I would suggest conforming to the same pattern as the "Lights" event to make it clear whether a value of "true" signifies that flight assist is on or off. Furthermore, I would suggest that we conform to the same name as the `status` object so that we're not dealing with properties being inverted or differing property names.
```cs
            VARIABLES.Add("flight_assist_off", "A boolean value. True if flight assist is off.");
```

Reply:
@Tkael Finally got around to it after the holidays. So, the reason I named this variable "status" was to avoid having the corresponding VoiceAttack variable named like `{BOOL:EDDI flight assist flight assist off}`.

I will change the name as per your request, but IMO it's ugly.

Reply:
I can live with `off`. The most important thing is to communicate which state it is in when true.